ajax/libs/e-modal/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "eModal",
```

Comment:
name in package.json should be parent folder name. In this case, my suggestion is that you can modify the folder name from e-modal to eModal.